libsel4vmmplatsupport/src/arch/arm/psci.c

```diff
@@ -50,15 +50,24 @@ int handle_psci(vm_vcpu_t *vcpu, seL4_Word fn_number, bool convention)
         smc_set_return_value(&regs, 0x00010000); /* version 1 */
         break;
     case PSCI_CPU_ON: {
-        uintptr_t target_cpu = smc_get_arg(&regs, 1);
+        uintptr_t requested_cpu = smc_get_arg(&regs, 1);
```

Comment:
so, my question holds - we get a (v)MPIDR here from the guest and not a linear (seL4 affinity) ID?